vulnerabilities/views.py

```diff
@@ -62,51 +63,99 @@ def get_purl_version_class(purl: models.Package):
     return purl_version_class
 
 
-class PackageSearch(ListView):
-    model = models.Package
-    template_name = "packages.html"
-    ordering = ["type", "namespace", "name", "version"]
+class BaseSearchView(ListView):
```

Comment:
Could we use the `django_filters.views.FilterView` instead of re-implementing it?